src/quo/components/wallet/amount_input/schema.cljs

```diff
@@ -0,0 +1,17 @@
+(ns quo.components.wallet.amount-input.schema)
+
+(def ?schema
```

Comment:
@ilmotta, @clauxx can you check this schema please?

Reply:
Hi, @ilmotta and @clauxx, I just wanted to ensure the request was not lost.
Thanks in advance!

Reply:
thanks for reminding, would've missed it :+1: